service/orchestrator/metrics.go

```diff
@@ -121,6 +121,8 @@ func prepareMetric(m *assessment.Metric) (err error) {
 	}
 
 	config.IsDefault = true
+	config.UpdatedAt = timestamppb.Now()
```

Comment:
I think the idea was that a default metric configuration has no update at (nil) because in fact it was never "updated" (from the default).